src/v2/Components/v2/ArtworkFilter/ArtworkFilters/ArtworkLocationFilter.tsx

```diff
@@ -0,0 +1,98 @@
+import { Checkbox, Flex, Text, Toggle } from "@artsy/palette"
```

Comment:
I'm hoping to generalize and improve the filters which have an autosuggest and long(er) list with the show/hide toggle. We have several now that can all be generalized and easily DRY'ed, but awaiting that final QA story (separate PR).

Reply:
Check out https://github.com/artsy/force/blob/master/src/v2/Components/v2/ArtworkFilter/ArtworkFilters/ShowMore.tsx